lib/pact_broker/matrix/aggregated_row.rb

```diff
@@ -33,7 +33,7 @@ def latest_verification_for_pseudo_branch
         @latest_verification ||= begin
           verification = matrix_rows.collect do | row|
               row.verification || latest_verification_for_consumer_version_tag(row)
-            end.compact.sort{ |v1, v2| v1.id <=> v2.id }.last
+            end.compact.sort_by(&:id).last
```

Comment:
Oh, every time I've done a sort, I've thought, I should work out if there's a symbol way to do it, and then I forget. Thanks!